test/fixtures/publisher_notes.yml

```diff
@@ -37,7 +37,7 @@ thread_note:
 
 child_note:
   publisher: just_notes
-  note: "This is child note"
+  note: "This is child note@"
```

Comment:
Unused change?

Reply:
No, this was related to one of the previewers which required a @ symbol.